Doc/Tutorial/chapter_quick_start.tex

```diff
@@ -16,7 +16,7 @@ \section{General overview of what Biopython provides}
 \section{Working with sequences}
 \label{sec:sequences}
 
-Disputably (of course!), the central object in bioinformatics is the sequence. Thus, we'll start with a quick introduction to the Biopython mechanisms for dealing with sequences, the \verb|Seq| object, which we'll discuss in more detail in Chapter~\ref{chapter:Bio.Seq}.
+Disputably (Of course!), the central object in bioinformatics is the sequence. Thus, we'll start with a quick introduction to the Biopython mechanisms for dealing with sequences, the \verb|Seq| object, which we'll discuss in more detail in Chapter~\ref{chapter:Bio.Seq}.
```

Comment:
http://english.stackexchange.com/questions/25232/exclamation-marks-in-the-middle-of-a-sentence cites the Chicago Manual of Style, 16th Edition *6.96 PARENTHESES WITH OTHER PUNCTUATION*. I don't have access to it to check the source, but I would agree this was informal writing with the exclamation mark for emphasis, so the capitalisation is not required.

However, this seems to be somewhat undefined, more a question of style than grammar.